package.json

```diff
@@ -6,7 +6,7 @@
     "@1hive/1hive-ui": "^1.0.2",
     "@1hive/connect-thegraph-conviction-voting": "^0.1.7",
     "@1hive/uniswap-sdk": "^1.0.0",
-    "@aragon/connect": "^0.3.0",
+    "@aragon/connect": "^0.5.0",
```

Comment:
I think we should upgrade this to the [latest patch version](https://github.com/aragon/connect/releases/tag/v0.5.2). :)

Reply:
Ah Pierre already did that here! https://github.com/1Hive/honey-pot/pull/36